docs/en/deployment/juicefs_on_docker.md

```diff
@@ -55,29 +55,137 @@ Run the following command to re-mount the JuiceFS file system with `allow_other`
 juicefs mount -d -o allow_other redis://<your-redis-url>:6379/1 /mnt/jfs
 ```
 
-## 2. Docker Volume Plugin
+## 2. Docker Volume Plugin {#docker-volume-plugin}
 
-[Volume plugin](https://docs.docker.com/engine/extend/) is another option to access JuiceFS.
+JuiceFS provides [volume plugin](https://docs.docker.com/engine/extend/) for Docker environments to create storage volumes on JuiceFS as if they were local disks.
 
-```sh
-$ docker plugin install juicedata/juicefs
-Plugin "juicedata/juicefs" is requesting the following privileges:
- - network: [host]
- - device: [/dev/fuse]
- - capabilities: [CAP_SYS_ADMIN]
-Do you grant the above permissions? [y/N]
+### Dependencies
+
+Since JuiceFS mount depends on FUSE, please make sure that the FUSE driver is already installed on the host, in the case of Debian/Ubuntu.
+
+```shell
+sudo apt-get -y install fuse
+```
+
+### Installation
+
+Install the Volume Plugin.
+
+```shell
+sudo docker plugin install juicedata/juicefs --alias juicefs
+```
+
+### Usage
+
+The process of creating a storage volume using the JuiceFS Docker Volume Plugin is similar to creating and mounting a filesystem in a Docker container using the JuiceFS client, so you need to provide information about the database and object storage so that the volume plugin can perform the appropriate operations.
 
-$ docker volume create -d juicedata/juicefs:latest -o name={{VOLUME_NAME}} -o metaurl={{META_URL}} -o access-key={{ACCESS_KEY}} -o secret-key={{SECRET_KEY}} jfsvolume
-$ docker run -it -v jfsvolume:/opt busybox ls /opt
+:::tip
+Since SQLite is a standalone database, the volume plugin container cannot read the database created by the host. Therefore, when using the Docker volume plugin, you can only use network based databases such as Reids, MySQL, etc.
+:::
+
+#### Create a Storage Volume
+
+Please replace `<VOLUME_NAME>`, `<META_URL>`, `<STORAGE_TYPE>`, `<BUCKET_NAME>`, `<ACCESS_KEY>`, `<SECRET_KEY>` in the following commands with your own.
+
+```shell
+sudo docker volume create -d juicefs \
+    -o name=<VOLUME_NAME> \
+    -o metaurl=<META_URL> \
+    -o storage=<STORAGE_TYPE> \
+    -o bucket=<BUCKET_NAME> \
+    -o access-key=<ACCESS_KEY> \
+    -o secret-key=<SECRET_KEY> \
+    jfsvolume
+```
+
+:::tip
+You can create multiple file systems on the same object storage bucket by specifying different `<VOLUME_NAME>` volume names and `<META_URL>` database address.
+:::
+
+For pre-created file systems, when creating volume plug-ins with them, simply specify the file system name and database address, e.g.
```

Comment:
To use docker volume plugin with existing JuiceFS volumes, simply specify the file system name and database address